src/components/Sidenav/SidenavMapUI.vue

```diff
@@ -10,6 +10,22 @@
       slot="tools"
       class="tools"
     >
+      <q-btn
```

Comment:
Ah, I found out why the link doesn't work! It seems that nesting `router-link` inside `q-btn` doesn't work. It has to be the other way round. Unfortunately, this creates a problem where the icon would disappear on hovering, so I'd propose to add a `click` handler to `q-btn` and remove the `router-link` completely:

```
@click="$router.push({name: 'map', params: {groupId: currentGroup.id}})"
```